README.md

```diff
@@ -144,4 +144,13 @@ To track how this government-funded code is used, we request that if you decide
 
 Digital Object Identifier: (fill in with DOI)
 
+To cite this code in a publication or report, please cite our associated report/paper and/or our source code:
+(Fill out the `formatted fields` in the below citation with details of your source code)
+
+`author_surname_or_organization`, `initial`. (`year`). _`program_or_source_code_title`_ (`code_version`) [Source code]. Provided by ITS CodeHub through GitHub.com. Accessed YYYY-MM-DD from `doi_url`.
```

Comment:
What initial is 'initial' supposed to be?

Reply:
Hm we could reword this. It's suppose to mean that when you list individual contributor names, it should follow the format of Last Name, First Name Initial. (e.g. Lien, C.).

Reply:
Yeah, it seems a bit confusing since we say they can use the organization name. At the least, I would name it 'first initial.'